lib/CL/devices/devices.c

```diff
@@ -339,6 +404,9 @@ pocl_uninit_devices ()
   cl_device_id d;
   for (i = 0; i < POCL_NUM_DEVICE_TYPES; ++i)
     {
+      if (pocl_devices_init_ops[i] == NULL) {
```

Comment:
Pls check style for a C file (the formatter script helps).

Reply:
I ran the formatter script `./tools/scripts/format-branch.sh`, but this file is not changed.

Reply:
LLVM style allows both, but I've changed the formatting to match the rest of the files.